x/issuer/keeper/keeper.go

```diff
@@ -189,7 +198,26 @@ func (k Keeper) AddIssuer(ctx sdk.Context, newIssuer types.Issuer) (*sdk.Result,
 	}
 
 	k.setIssuers(ctx, issuers)
-	k.ik.AddDenoms(ctx, newIssuer.Denoms) // TODO Check error?
+	for _, denom := range newIssuer.Denoms {
+		if stDenom := k.bk.GetDenomMetaData(ctx, denom); stDenom.Base == "" {
+			k.bk.SetDenomMetaData(ctx, banktypes.Metadata{
+				Description: "e-Money EUR stablecoin",
+				DenomUnits: []*banktypes.DenomUnit{
+					{
+						Denom:    denom,
+						Exponent: 6,
+						Aliases:  nil,
+					},
+				},
+				Base:    denom,
+				Display: strings.ToUpper(denom),
```

Comment:
Would also be nice to have this parameterizable. Optional flags would be nice here, e.g. `emd tx ... --display "eEUR" --description "THE Euro stablecoin"`. If omitted, the description should be empty, while the `display` value should be the uppercased denom (as it is now).